server/routes/routerRequestHandlers/createEmptyInductionIfNotInSession.test.ts

```diff
@@ -36,15 +36,7 @@ describe('createEmptyInductionIfNotInSession', () => {
     // Given
     req.session.inductionDto = undefined
 
-    const educationServiceError = {
-      status: 404,
-      data: {
-        status: 404,
-        userMessage: `Education not found for prisoner [${prisonNumber}]`,
-        developerMessage: `Education not found for prisoner [${prisonNumber}]`,
-      },
-    }
-    educationAndWorkPlanService.getEducation.mockRejectedValue(educationServiceError)
+    educationAndWorkPlanService.getEducation.mockResolvedValue(undefined)
```

Comment:
The basic change is that the service layer now checks the status code from the client response. If it's a 404 then it returns undefined, otherwise it continues to throw back the error.